### ISSUE TITLE
Why part of the deck has no color when I checked in the ifc file?

### ISSUE THREAD

**Issue by lixy-toone** [opensourceBIM/BIMsurfer-before2019#206]
The first picture is the ifc file showed by the BIMsurfer.
![hzmb](https://user-images.githubusercontent.com/28996034/26865957-ae388208-4b91-11e7-8d9b-dd2b236b703f.png)

The second one is the rvt file showed by the Revit.
![hzmb_ori](https://user-images.githubusercontent.com/28996034/26866052-1304a0ea-4b92-11e7-9bbb-0cdf59699a95.png)

Why part of the deck has no color showed by the BIMsurfer?

**Comment by rubendel** [comment]
Can you share the file?

**Comment by lixy-toone** [comment]
Yes.But it is almost 700MB.How can I send it to you?Could you give me a contract way?

**Comment by rubendel** [comment]
Cool, https://www.dropbox.com/request/J4rYO1rxnn0SjBWo9VRp

**Comment by lixy-toone** [comment]
My network can't connect to dropbox.Do you have QQ and MSN? And , can you visit http://pan.baidu.com/?

**Comment by lixy-toone** [comment]
I have uploaded the ifc file on Baidu Netdisk that you can download it in the webpage http://pan.baidu.com/s/1hr2SprI.
Try it, please!

**Comment by rubendel** [comment]
Hi sorry this asks me to install an executable, please find another way. I do not use QQ nor MSN.

**Comment by lixy-toone** [comment]
Do you use Skype?

**Comment by lixy-toone** [comment]
Hi rubendel, 
I have uploaded my ifc file on your dropbox successfully .Please check it.

**Comment by lixy-toone** [comment]
Hi rubendel,
Did you download the ifc file and check it?

**Comment by rubendel** [comment]
Yes, but so far I have no clue as to why the deck has a different color there.

**Comment by lixy-toone** [comment]
A part of the deck seems transparent.I wish it were gray.
Could you study with me?We can set little bit color on it probably.

**Comment by lixy-toone** [comment]
![20170610-1](https://user-images.githubusercontent.com/28996034/27001040-d0d491cc-4df2-11e7-9ffb-4e38ab068faf.png)
When the ifc file was being checked in by the bimserver, during importing two part of the deck had something wrong.So the google chrome throwed two errors, when the bridge was showed by the bimsurfer.
Could you find the reason why the two part of the deck were imported failure?
Are there any bugs in bimserver?
